wpi-gpio.js

```diff
@@ -58,6 +58,36 @@
   };
 
   /**
+   * Set pull up resistor on a pin
+   *
+   * @param {Number|String} pin
+   * @return {Promise}
+   */
+  gpio.pullUp = function(pin) {
+    return gpioExec('mode', pin, ['up']);
+  };
+
+  /**
+   * Set pull down resistor on a pin
+   *
+   * @param {Number|String} pin
+   * @return {Promise}
+   */
+  gpio.pullDwon = function(pin) {
```

Comment:
I have and i am testing it right now in my house. But now i start to think internal pull-up pull-down are not good idea since interference from relays is causing them to detect false edges probably because they are too week :D